### PR TITLE
Added a macro that prints the Elementary profile structure.

### PR DESCRIPTION
**Pull request by elongl**
Upon installation, the user runs `dbt run-operation elementary.print_elementary_profile` to get the `elementary` profile.

### CODE REVIEW

File: macros/edr/system/configuration/print_elementary_cli_profile.sql [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{% macro print_elementary_cli_profile() %}
+    {% set elementary_database, elementary_schema = get_package_database_and_schema() %}
```
Comment (oravi) [inline macros/edr/system/configuration/print_elementary_cli_profile.sql:2]: Use elementary prefix

File: macros/edr/system/configuration/print_elementary_cli_profile.sql [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{% macro print_elementary_cli_profile() %}
+  {% set elementary_database, elementary_schema = elementary.get_package_database_and_schema() %}
+  {{ log('\n' ~ adapter.dispatch('print_elementary_profile')(elementary_database, elementary_schema), info=True) }}
+{% endmacro %}
+
+{% macro snowflake__print_elementary_profile(elementary_database, elementary_schema) %}
```
Comment (oravi) [inline macros/edr/system/configuration/print_elementary_cli_profile.sql:6]: It's also better to have a consistent name with the macro above

**Review by Maayan-s — COMMENTED** [review]
As it's an operation for users (not an internal function), I would change the name. 
Think about the step in the onboarding:
"After you finish installing the CLI, run the operation 'print_elementary_cli_profile' to generate a connection profile that points to the monitored dbt project." 
This would be better:
"After you finish installing the CLI, run the operation 'generate_elementary_cli_profile', that will be used to point CLI to the monitored dbt project."

**Comment by Maayan-s** [comment]
Another question I had - does it work in dbt cloud?

**Comment by elongl** [comment]
> As it's an operation for users (not an internal function), I would change the name. Think about the step in the onboarding: "After you finish installing the CLI, run the operation 'print_elementary_cli_profile' to generate a connection profile that points to the monitored dbt project." This would be better: "After you finish installing the CLI, run the operation 'generate_elementary_cli_profile', that will be used to point CLI to the monitored dbt project."

What worries me is that calling it `generate` might convey that we create a new one on each invocation.
When I call it `print`, it's clearer that it's solely about printing the Elementary profile back to the user.
What do you think? @Maayan-s

**Comment by elongl** [comment]
> Another question I had - does it work in dbt cloud?

It should, what makes you think that it won't?

**Comment by elongl** [comment]
> Another question I had - does it work in dbt cloud?

No. `profiles.yml` is a Dbt CLI concept.
image

However, the macro itself would still work which would help users to define their profile in Dbt Cloud.

**Comment by Maayan-s** [comment]
> > Another question I had - does it work in dbt cloud?
> 
> No. `profiles.yml` is a Dbt CLI concept.
> 
> image
> 
> However, the macro itself would still work which would help users to define their profile in Dbt Cloud.

Yes I was asking if all these jinja refs are available in cloud although it does not support profiles

**Comment by Maayan-s** [comment]
> > As it's an operation for users (not an internal function), I would change the name. Think about the step in the onboarding: "After you finish installing the CLI, run the operation 'print_elementary_cli_profile' to generate a connection profile that points to the monitored dbt project." This would be better: "After you finish installing the CLI, run the operation 'generate_elementary_cli_profile', that will be used to point CLI to the monitored dbt project."
> 
> What worries me is that calling it `generate` might convey that we generate a new one on each invocation. When I call it `print`, it's clearer that it's solely about printing the Elementary profile back to the user. What do you think? @Maayan-s

We don't print an existing profile, we do generate one. No?
What do you mean by "generate a new one on each invocation"?

**Comment by elongl** [comment]
> > > Another question I had - does it work in dbt cloud?
> > 
> > 
> > No. `profiles.yml` is a Dbt CLI concept.
> > image
> > However, the macro itself would still work which would help users to define their profile in Dbt Cloud.
> 
> Yes I was asking if all these jinja refs are available in cloud although it does not support profiles

Tested, it does work on Dbt cloud.

**Pull request merged**